deploy/helm/templates/addons/nebula-addon.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions.kubeblocks.io/v1alpha1
+kind: Addon
+metadata:
+  name: nebula
+  labels:
+    {{- include "kubeblocks.labels" . | nindent 4 }}
+    "kubeblocks.io/provider": community
+  {{- if .Values.keepAddons }}
+  annotations:
+    helm.sh/resource-policy: keep
+  {{- end }}
+spec:
+  description: 'NebulaGraph is an Apache 2.0 licensed distributed graph database.'
+
+  type: Helm
+
+  helm:
+    {{- if hasPrefix "oci://" .Values.addonChartLocationBase }}
+    chartLocationURL: {{ .Values.addonChartLocationBase }}/nebula
+    {{- else }}
+    chartLocationURL: {{ .Values.addonChartLocationBase }}/nebula-{{ default .Chart.Version .Values.versionOverride }}.tgz
+    {{- end }}
+
+    installOptions:
+     {{- if hasPrefix "oci://" .Values.addonChartLocationBase }}
+      version: {{ default .Chart.Version .Values.versionOverride }}
+     {{- end }}
+
+  installable:
+    autoInstall: true
```

Comment:
set `autoInstall` to `false`. We can enable this addon on demand later using `kbcli`.